packages/create-pantheon-decoupled-kit/src/types.ts

```diff
@@ -53,6 +53,10 @@ export interface DecoupledKitGenerator<
 	 * Any message(s) to be rendered after actions are successfully completed.
 	 */
 	nextSteps?: string[];
+	/**
+	 * Identifies a generators compatible CMS(s).
+	 */
+	cmsType: string;
```

Comment:
This type can be more specific since we only have a specific set of cmsTypes
```suggestion
	cmsType: 'wp' | 'drupa' | 'any';
```